src/systemctl/systemctl.c

```diff
@@ -85,6 +85,23 @@
 #include "verbs.h"
 #include "virt.h"
 
+/* The init script exit status codes
+   0       program is running or service is OK
+   1       program is dead and /var/run pid file exists
+   2       program is dead and /var/lock lock file exists
+   3       program is not running
+   4       program or service status is unknown
+   5-99    reserved for future LSB use
+   100-149 reserved for distribution use
+   150-199 reserved for application use
+   200-254 reserved
+*/
```

Comment:
Hmm, so we already have src/basic/exit-status.h and it defines many LSB exit codes already. However, if defines the ones that are defined for the LSB "status" command, not the ones for the LSB "start" command. And I am pretty sure that it's the one for "status" that matter here, not the ones for "start"...

i.e. the values from the bottom of

http://refspecs.linuxfoundation.org/LSB_3.2.0/LSB-Core-generic/LSB-Core-generic/iniscrptact.html

not the ones at the top...